_layouts/indicator.html

```diff
@@ -84,9 +84,10 @@ <h4>{{ t.indicator.sub_categories }}</h4>
             <li role="presentation" class="nav-item map" style="display:none">
               <a data-no-disagg="true" class="nav-link" data-toggle="tab" href="#mapview" aria-controls="mapview" role="tab" {% include autotrack.html preset="tab_data_map" category="Tab change" action="Change data view" label="Change to Map tab" %}>{{ t.indicator.map }}</a>
             </li>
-            {%- if meta.embedded_map_html -%}
+            {%- if meta.embedded_feature_url -%}
```

Comment:
I like the change. Just noting that we should call this out as a breaking change in the release notes.